robottelo/ui/contenthost.py

```diff
@@ -0,0 +1,91 @@
+"""Utilities to manipulate content hosts via UI."""
+from robottelo.ui.base import Base, UIError
+from robottelo.ui.locators import common_locators, locators, tab_locators
+from robottelo.ui.navigator import Navigator
+
+
+class ContentHost(Base):
+    """Manipulates Content Hosts from UI"""
+    is_katello = True
+
+    def navigate_to_entity(self):
+        """Navigate to Content Hosts entity page"""
+        Navigator(self.browser).go_to_content_hosts()
+
+    def _search_locator(self):
+        """Specify locator for Content Hosts entity search procedure"""
+        return locators['contenthost.select_name']
+
+    def add_subscriptions(self, subscriptions=None, tab_locator=None,
+                          select_locator=None):
+        """Add or remove subscription association for content host."""
+        strategy, value = locators['contenthost.subscription_select']
+        self.click(tab_locators['contenthost.tab_subscriptions'])
+        self.click(tab_locators['contenthost.tab_subscriptions_subscriptions'])
+        if not self.find_element(tab_locator):
```

Comment:
I suggest to use `self.wait_until_element` instead `find_element` to have more stable runs in future